packages/studio/styled-system/types/style-props.d.ts

```diff
@@ -8,8 +8,8 @@ type AnyString = (string & {})
 type CssVars = `var(--${string})`
 type CssVarValue = ConditionalValue<Token | AnyString | (number & {})>
 
-type CssVarName =  | AnyString
-type CssVarKeys = `--${CssVarName}`
+type CssVarName = 
```

Comment:
How can I prevent this empty type? I tried to fix this by editting `style-props.ts`, but it didn't work.